src/Tasks/Microsoft.NET.Build.Tasks/FileResolver.cs

```diff
@@ -10,23 +10,23 @@
 
 namespace Microsoft.NET.Build.Tasks
 {
-    internal class PublishAssembliesResolver
+    internal class FileResolver
```

Comment:
FileResolver is a rather generic name, perhaps something like AssetsFileResolver would be better.

Reply:
I'll change.